packages/calling-stateful-client/src/CallAgentDeclarative.test.ts

```diff
@@ -108,7 +116,7 @@ class MockCallAgentWithMultipleCalls implements CallAgent {
   calls: MockCall[] = [];
   displayName = undefined;
   emitter = new EventEmitter();
-
+  feature;
```

Comment:
ditto: looks like leaked from earlier iteration of the PR.

Reply:
Let me double check on this one I remember there was some wierd feature property added so ill double check on this one and either clean it up or comment further on this.

Reply:
I think we may need this feature property. It was added to CallAgent and we happen to have two mocks of CallAgent for our tests.

`> @internal/calling-stateful-client@1.0.1-beta.2 build:esm /workspaces/communication-ui-library/packages/calling-stateful-client
> if-env COMMUNICATION_REACT_FLAVOR=stable && rushx preprocess && tsc -project tsconfig.preprocess.json || (if-env COMMUNICATION_REACT_FLAVOR=beta && tsc)

src/CallAgentDeclarative.test.ts:115:7 - error TS2420: Class 'MockCallAgentWithMultipleCalls' incorrectly implements interface 'CallAgent'.
  Property 'feature' is missing in type 'MockCallAgentWithMultipleCalls' but required in type 'CallAgent'.

115 class MockCallAgentWithMultipleCalls implements CallAgent {
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  ../../common/temp/node_modules/.pnpm/@azure/communication-calling@1.4.2-beta.1/node_modules/@azure/communication-calling/types/communication-calling-beta.d.ts:358:5
    358     feature<TFeature extends CallAgentFeature>(factory: CallAgentFeatureFactory<TFeature>): TFeature;
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    'feature' is declared here.

src/CallAgentDeclarative.test.ts:345:55 - error TS2345: Argument of type 'MockCallAgentWithMultipleCalls' is not assignable to parameter of type 'CallAgent'.
  Property 'feature' is missing in type 'MockCallAgentWithMultipleCalls' but required in type 'CallAgent'.

345     const declarativeCallAgent = callAgentDeclaratify(mockCallAgent, context, internalContext);
                                                          ~~~~~~~~~~~~~

  ../../common/temp/node_modules/.pnpm/@azure/communication-calling@1.4.2-beta.1/node_modules/@azure/communication-calling/types/communication-calling-beta.d.ts:358:5
    358     feature<TFeature extends CallAgentFeature>(factory: CallAgentFeatureFactory<TFeature>): TFeature;
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    'feature' is declared here.`
![image](https://user-images.githubusercontent.com/79329532/154754623-4cd6f109-8fe6-491c-97be-a230d69ed344.png)